src/tools/plotter/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(iDynTreePlotter)
+
+find_package(Qt5 COMPONENTS Widgets Charts)
+if (${Qt5_FOUND})
+  set(CMAKE_CXX_STANDARD 11)
```

Comment:
This is effective only on CMake >= 3.1 . I suggest to use the `idyntree_enable_cxx11` macro instead. https://github.com/robotology/idyntree/blob/devel/cmake/iDynTreeOptions.cmake#L107`